style/feed.less

```diff
@@ -0,0 +1,110 @@
+.feed {
+	background-color: #f5f5f5;
+	position: absolute;
+	width: 100vw;
+
+	&.collapsed {
+		bottom: 30px;
+		height: 50px;
+		border-radius: 6px 6px 0 0;
+
+		.header {
+			margin: 0 31px 0 11px;
+		}
+
+		.list {
+			top: 32px;
+			bottom: 0;
+
+			.item {
+				height: 45px;
+				background-color: #f5f5f5;
+			}
+		}
+	}
+
+	&.expanded {
+		height: 255px;
+
+		.list {
+			top: 33px;
+			bottom: 0;
+		}
+	}
+
+	.header {
+		width: 73px;
+		height: 14px;
+		margin: 10px 31px 9px 11px;
+		opacity: 0.9;
+		text-transform: uppercase;
+		font-size: 12px;
+		font-weight: bold;
+		letter-spacing: 1px;
+		color: #201212;
+		background-color: #f5f5f5;
+		padding: 0;
+	}
+
+	.cue {
+		width: 15px;
+		height: 3px;
+		margin: auto;
+		margin-top: 8px;
+		border-radius: 1.5px;
+		background-color: #ccc;
+	}
+
+	.list {
```

Comment:
<ListView> already has its own styling consistent across the app. Wondering why we repeat some of it here and it's also slightly different, especially the selected item.

Reply:
You're right I do need to clean up the repeated rules. I'm overriding the slightly different parts in feed.less so it fits what design is requesting